mm/kasan/common.c

```diff
@@ -493,6 +500,7 @@ static void *__kasan_kmalloc(struct kmem_cache *cache, const void *object,
 	if (cache->flags & SLAB_KASAN)
 		set_track(&get_alloc_info(cache, object)->alloc_track, flags);
 
+out:
 	return set_tag(object, tag);
```

Comment:
What happens if we set a tag?

Reply:
Hm, actually, we'd better skip this path. It doesn't matter for KASAN, but KHWASAN is a different story.

Reply:
Removed the goto